ws-nextjs-app/integration/pages/av-embeds/canonicalTests.ts

```diff
@@ -0,0 +1,7 @@
+import { runMediaPlayerTests } from '#src/integration/common';
+import runResponseHeaderTests from '#nextjs/integration/utils/responseHeaderTests';
+
+export default () => {
+  runResponseHeaderTests();
```

Comment:
Is this wired into the Live page integration suite? Not sure if Live pages might not be covered by this req-svc-chain work